### PR TITLE
Installation Guide: Setup files (server and proxy)

### PR DESCRIPTION
**Pull request by keichwa**
Setup files, starting with proxy.
https://github.com/SUSE/spacewalk/issues/8363

### CODE REVIEW

File: modules/installation/pages/proxy-registration.adoc [hunk 1]
```diff
@@ -58,10 +54,13 @@ xref:client-configuration:registration-bootstrap.adoc[].
 image::proxy-bootstrap.png[]
 
 . Create the SUSE Manager Tools Repository for bootstrapping, see xref:client-configuration:creating-a-tools-repository.adoc[Create Tools Repository].
-. Bootstrap the client with the bootstrap script.  For more information, see xref:client-configuration:registration-bootstrap.adoc[].
-. In case of a Salt client, accept the key on the menu:Main Menu[Salt > Keys] page by clicking the check mark.
-It then will appear in the menu:Main Menu[Systems > Overview].
-. Check via menu:System Details[Software > Software Channels] that the four proxy channels ([systemitem]``Pool`` and [systemitem]``Updates`` for [systemitem]``SLE-PRODUCT`` and [systemitem]``SLE-MODULE``) plus the recommended channels are selected; [systemitem]``SLE-PRODUCT-Pool`` as the base channel and the others as child channels.
+. Bootstrap the client with the bootstrap script.
+For more information, see xref:client-configuration:registration-bootstrap.adoc[].
+. In case of a Salt client, accept the key on the menu:Salt[Keys] page by clicking the check mark.
```
Comment (Loquacity) [inline modules/installation/pages/proxy-registration.adoc]:
```suggestion
. For Salt clients, accept the key on the menu:Salt[Keys] page by checking the appropriate checkbox.
```

File: modules/installation/pages/proxy-registration.adoc [hunk 2]
```diff
@@ -58,10 +54,13 @@ xref:client-configuration:registration-bootstrap.adoc[].
...
+When accepted, it will appear in the menu:Systems[Overview].
+. Check via menu:System Details[Software > Software Channels] that the four proxy channels ([systemitem]``Pool`` and [systemitem]``Updates`` for [systemitem]``SLE-PRODUCT`` and [systemitem]``SLE-MODULE``) plus the recommended channels are selected.
```
Comment (Loquacity) [inline modules/installation/pages/proxy-registration.adoc]:
```suggestion
. Navigate to menu:System Details[Software > Software Channels], and check that the four proxy channels ([systemitem]``Pool`` and [systemitem]``Updates`` for [systemitem]``SLE-PRODUCT`` and [systemitem]``SLE-MODULE``) plus the recommended channels are selected.
```

File: modules/installation/pages/proxy-setup.adoc [hunk 3]
```diff
@@ -1,11 +1,7 @@
 [[proxy-setup]]
 = SUSE Manager Proxy Setup
 
-
-
-
-
-{productname}  Proxy requires additional configuration in order to make it useful.
+{productname} Proxy requires additional configuration to make it useful.
```
Comment (Loquacity) [inline modules/installation/pages/proxy-setup.adoc]:
```suggestion
{productname} Proxy requires additional configuration.
```

File: modules/installation/pages/proxy-setup.adoc [hunk 4]
```diff
@@ -22,25 +18,27 @@ The new salt-broker service will be automatically started at the end of the pack
 This service forwards the Salt interactions to the {productname} server.
 ////
 
+// REMARK 2019-08-23, ke: Is this a proxy-only feature?
 [NOTE]
 .Proxy Chains
 ====
 It is possible to arrange Salt proxies in a chain.
 In such a case, the upstream proxy is named "`parent`".
```
Comment (Loquacity) [inline modules/installation/pages/proxy-setup.adoc]:
```suggestion
In such a case, the upstream proxy is named `parent`.
```

File: modules/installation/pages/proxy-setup.adoc [hunk 5]
```diff
@@ -22,25 +18,27 @@ The new salt-broker service will be automatically started at the end of the pack
...
 ====
 
-Make sure the proxie's TCP ports `4505` and `4506` are open and that the proxy can reach the {productname} server (or another upstream proxy) on these ports.
+Make sure the proxie's TCP ports `4505` and `4506` are open.
```
Comment (Loquacity) [inline modules/installation/pages/proxy-setup.adoc]:
```suggestion
Make sure the TCP ports `4505` and `4506` are open on the proxy.
```

File: modules/installation/pages/proxy-setup.adoc [hunk 6]
```diff
@@ -104,7 +102,7 @@ For further information on country codes, refer to the online https://www.iso.or
 [TIP]
 .Country Code
 ====
-As the country code enter the country code set during the SUSE Manager installation.
+As the country code enter the country code set during the {productname} installation.
```
Comment (Loquacity) [inline modules/installation/pages/proxy-setup.adoc]:
```suggestion
In the [guimenu]``country code`` field, enter the country code set during the {productname} installation.
```

File: modules/installation/pages/proxy-setup.adoc [hunk 7]
```diff
@@ -104,7 +102,7 @@ For further information on country codes, refer to the online https://www.iso.or
...
 For example, if your proxy is in US and your {productname} in DE, you must enter `DE` for the proxy.
```
Comment (Loquacity) [inline modules/installation/pages/proxy-setup.adoc]:
```suggestion
For example, if your proxy is in the US and your {productname} is in DE, enter `DE` for the proxy.
```

File: modules/installation/pages/proxy-setup.adoc [hunk 8]
```diff
@@ -129,35 +127,35 @@ Activate advertising proxy via SLP?::
 SLP stands for Service Location Protocol.
 
 If parts are missing, such as CA key and public certificate, the script prints commands that you must execute to integrate the needed files.
-When the mandatory files are copied, re-run [command]``configure-proxy.sh``.
-Also restart the script if a HTTP error was met during script execution.
+When the mandatory files are copied, run [command]``configure-proxy.sh`` again.
+Also run again the script if a HTTP error was met during script execution.
```
Comment (Loquacity) [inline modules/installation/pages/proxy-setup.adoc]:
```suggestion
If you receive an HTTP error during script execution, run the script again.
```

File: modules/installation/pages/proxy-setup.adoc [hunk 9]
```diff
@@ -129,35 +127,35 @@ Activate advertising proxy via SLP?::
...
-[guimenu]``Connection`` and [guimenu]``Proxy`` subtabs display the respective status information.
+To check the status of the proxy system and its clients, click the proxy system's details page on the {webui} (menu:Systems[Proxy], then the system name).
+[guimenu]``Connection`` and [guimenu]``Proxy`` subtabs display various status information.
 
 
 
 
 [[proxy.pxe.setup]]
-== Enabling PXE Boot via {susemgrproxy}
+== Enable PXE Boot through {susemgrproxy}
```
Comment (Loquacity) [inline modules/installation/pages/proxy-setup.adoc]:
```suggestion
== Enable PXE Boot
```

File: modules/installation/pages/proxy-setup.adoc [hunk 10]
```diff
@@ -129,35 +127,35 @@ Activate advertising proxy via SLP?::
...
-. On the {productname} server install [package]#susemanager-tftpsync# :
+. On the {productname} Server install [package]#susemanager-tftpsync#:
 +
 
 ----
 zypper in susemanager-tftpsync
 ----
 
-. On the {susemgrproxy} server install [package]#susemanager-tftpsync-recv# :
+. On the {susemgrproxy} Server install [package]#susemanager-tftpsync-recv#:
```
Comment (Loquacity) [inline modules/installation/pages/proxy-setup.adoc]: SUSE Manager Proxy Server?

File: modules/installation/pages/proxy-setup.adoc [hunk 11]
```diff
@@ -209,8 +208,10 @@ filename: "pxelinux.0"
 ----
 
 
+
+// REMARK: 2019-08-23, ke: this needs closer checking
 [[replacing.a.susemgrproxy]]
-== Replacing a {susemgrproxy}
+== Replace a {susemgrproxy}
 
 A {susemgrproxy} is `dumb` in the sense that it does not contain any information about the clients that are connected to it.
```
Comment (Loquacity) [inline modules/installation/pages/proxy-setup.adoc]:
```suggestion
A {susemgrproxy} is dumb in that it does not contain any information about the clients that are connected to it.
```

File: modules/installation/pages/server-setup.adoc [hunk 12]
```diff
@@ -62,13 +64,12 @@ image::quickstart-mgr-setup5-ncurses.png[scaledwidth=80%]
 
 . Click btn:[Yes] to run setup when prompted.
 
-. Once setup has completed, click btn:[Next] to continue.
+. When the setup has completed, click btn:[Next] to continue.
```
Comment (Loquacity) [inline modules/installation/pages/server-setup.adoc]:
```suggestion
. When setup is complete, click btn:[Next] to continue.
```

File: modules/installation/pages/server-setup.adoc [hunk 13]
```diff
@@ -139,15 +142,15 @@ For more information, see xref:reference:admin/setup-wizard.adoc[Wizard].
...
 image::admin_suse_products.png[scaledwidth=80%]
-+
 
 . If your {sle} client is based on [systemitem]``x86_64`` architecture scroll down the page and select the check box for this channel now.
++
 
 * Add channels to {productname} by selecting the check box to the left of each channel.
 Click the arrow symbol to the left of the description to unfold a product and list available modules.
-* Start product synchronization by clicking the btn:[Add Products] button.
+* Start product synchronization by clicking btn:[Add Products].
```
Comment (Loquacity) [inline modules/installation/pages/server-setup.adoc]:
```suggestion
* Click btn:[Add Products] to start product synchronization.
```

File: modules/installation/pages/server-setup.adoc [hunk 14]
```diff
@@ -139,15 +142,15 @@ For more information, see xref:reference:admin/setup-wizard.adoc[Wizard].
...
 
-After adding the channel {productname} will schedule the channel to be copied.
+After adding the channel {productname} will schedule the channel to be synchronized.
```
Comment (Loquacity) [inline modules/installation/pages/server-setup.adoc]:
```suggestion
After adding the channel, {productname} will schedule the channel to be synchronized.
```

File: modules/installation/pages/server-setup.adoc [hunk 15]
```diff
@@ -167,10 +170,10 @@ To write the new configuration run [command]``grub2-mkconfig -o /boot/grub2/grub
...
 
-Monitor channel synchronization process in real-time by viewing channel log files located in the directory [path]``/var/log/rhn/reposync``:
+Monitor the channel synchronization process in real-time by viewing channel log files located in the directory [path]``/var/log/rhn/reposync``:
 
 ----
 tail -f /var/log/rhn/reposync/<CHANNEL_NAME>.log
 ----
 
-After the channel sync process has completed proceed to xref:client-configuration:manual-registration-overview.adoc[Client Registration].
+After the channel synchronization process has completed continue with xref:client-configuration:manual-registration-overview.adoc[Client Registration].
```
Comment (Loquacity) [inline modules/installation/pages/server-setup.adoc]:
```suggestion
When the channel synchronization process is complete, you can continue with client registration.
For more instructions, see xref:client-configuration:manual-registration-overview.adoc[].
```

File: modules/installation/pages/proxy-setup.adoc (hunk 10 above)
Comment (keichwa) [reply modules/installation/pages/proxy-setup.adoc]: We sometimes use (or used) to call the proxy a "proxy server"--this is probably confusing and we should stop it... @Loquacity

File: modules/installation/pages/proxy-setup.adoc [hunk 16]
```diff
@@ -22,25 +18,27 @@ The new salt-broker service will be automatically started at the end of the pack
 This service forwards the Salt interactions to the {productname} server.
 ////
 
+// REMARK 2019-08-23, ke: Is this a proxy-only feature?
```
Comment (keichwa) [inline modules/installation/pages/proxy-setup.adoc]:
```suggestion
// REMARK 2019-08-23, ke: Is this a Salt proxy-only feature?
```

File: modules/installation/pages/proxy-setup.adoc (hunk 10 above)
Comment (Loquacity) [reply modules/installation/pages/proxy-setup.adoc]: yeah, it doesn't take much to miss the word 'proxy' in there.

Comment (keichwa) [reply modules/installation/pages/proxy-setup.adoc]: ok, please review again.

File: modules/installation/pages/proxy-setup.adoc [hunk 17]
```diff
@@ -224,12 +224,12 @@ If you do not use the reactivation key, you will need to re-registered all the c
...
 .Procedure: Replacing a {susemgrproxy} and Keeping the Clients Registered
 . Before starting the actual migration procedure, save the data from the old proxy, if needed.
-Consider copying important data to a central place that can also be accessed by the new proxy server.
+Consider copying important data to a central place that can also be accessed by the new proxy.
 // REMARK: 2019-08-23, ke: this is unclear
 // ** Copy the scripts that are still needed.
 // ** Copy the activation keys from the previous server.
 // Of course, it is always better to re-create the keys.
-. Shutdown the proxy server.
+. Shutdown the proxy.
```
Comment (Loquacity) [inline modules/installation/pages/proxy-setup.adoc]: Shut down.

**Pull request merged**